custom/enikshay/reports/repeaters.py

```diff
@@ -32,6 +32,7 @@ def _get_repeaters(self):
 class ENikshayForwarderReport(DomainForwardingRepeatRecords):
     name = 'eNikshay Forwarder Report'
     base_template = 'reports/base_template.html'
+    asynchronous = True
```

Comment:
I think this is the golden ticket... maybe this is obvious, but do `emailable` reports also have to be marked `asynchronous`?